microsoft/testsuites/performance/common.py

```diff
@@ -47,7 +47,7 @@ def perf_disk(
     test_name: str = "",
     num_jobs: Optional[List[int]] = None,
     block_size: int = 4,
-    time: int = 120,
+    time: int = 300,
```

Comment:
Same as comments in ntttcp.

Reply:
The file is specific to performance tests unlike earlier ntttcp changes where my changes were at tool level. I think its okay to change the time here.

Reply:
The number is well tuned for perf measurement, it's balanced between cost and identifying issues. This value can be used for all performance purposes tests. If it needs to extend longer in stress or longhaul tests, it can be extended in test case level.